modules/tooltip/react/stories/stories.tsx

```diff
@@ -45,7 +45,7 @@ class TooltipWrapper extends React.Component<{}, TooltipWrapperState> {
             aria-label="Close"
           />
         </div>
-        <Popper open={open} anchorEl={this.state.anchorEl} placement={'top'}>
+        <Popper open={open} anchorElement={this.state.anchorEl} placement={'top'}>
```

Comment:
These changes here will conflict with my Tooltip changes, but I can deal with that merge conflict